README.md

```diff
@@ -1,2 +1,131 @@
 # node-setup
+
 Connected Cars JavaScript/TypeScript development setup
+
+## Use
+
+``` bash
+npm install https://github.com/connectedcars/node-setup
+```
+
+### Automatic
+
+Don't overwrite existing files:
+
+``` bash
+./node_modules/.bin/setup init
+```
+
+Overwrite and remove old configurations:
+
+``` bash
+./node_modules/.bin/setup init-force
```

Comment:
Wouldn't it be more standardized to use `init --force`?

Reply:
Yes, I was too lazy to do argument parsing.